Cargo.toml

```
@@ -20,7 +20,7 @@ log = "0.4"
 regex = "1"
 structopt = "0.3"
 crates-index = "0.15.1"
-crates-index-diff = "7"
+crates-index-diff = "7.1"
```

Looks like the new feature was broken in 7.1.0. https://github.com/Byron/crates-index-diff-rs/blob/master/changelog.md#v711-2020-05-23

```suggestion
crates-index-diff = "7.1.1"
```